### ISSUE TITLE
Identical table names in different schemas are merged during introspection

### ISSUE THREAD

**Issue by RobinBlomberg** [koskimas/kysely#129]
If you have the following PostgreSQL tables:
```sql
CREATE TABLE "foo"."bar" ("fizz" timestamp);
CREATE TABLE "qux"."bar" ("buzz" numeric);
```
They get incorrectly introspected as (paraphrased):
```ts
[
  { schema: 'foo', name: 'bar', columns: [
    { name: 'fizz', dataType: 'timestamp' },
    { name: 'buzz', dataType: 'numeric' },
  ] },
  { schema: 'qux', name: 'bar', columns: [
    { name: 'fizz', dataType: 'timestamp' },
    { name: 'buzz', dataType: 'numeric' },
  ] },
]

**Comment by koskimas** [comment]
Nice catch! I'll release a new version soon that has this fixed.

**Comment by koskimas** [comment]
Fixed in 0.20.0
